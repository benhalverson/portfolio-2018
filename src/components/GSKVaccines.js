import React, { Fragment } from 'react';

import GSK001 from '../images/GSK001.png';
import GSK002 from '../images/GSK002.png';
import GSK003 from '../images/GSK003.png';
import GSK004 from '../images/GSK004.png';
import GSK005 from '../images/GSK005.png';

const GSKVaccines = () => (
  <Fragment>
    <div className="title">GSK Vaccines Eduaction Module</div>
    <div className="body">
      <p>
        This was a Web app based on an education framework called Adapt
        Learning. It was selected because of a really short deadline given by
        the client. The technology it used was Backbone.js and SCSS.
      </p>
      <div className="screenshots">
        <img className="col" src={GSK001} alt="" />
        <img className="col" src={GSK002} alt="" />
        <img className="col" src={GSK003} alt="" />
        <img className="col" src={GSK004} alt="" />
        <img className="col" src={GSK005} alt="" />
      </div>
    </div>
  </Fragment>
);

export default GSKVaccines;
