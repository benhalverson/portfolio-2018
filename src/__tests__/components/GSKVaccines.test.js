import React from 'react';
import { shallow } from 'enzyme';
import GSKVaccines from '../../components/GSKVaccines';
test('it should render the GSKVaccines component', () => {
  const wrapper = shallow(<GSKVaccines/>);
  expect(wrapper).toMatchSnapshot();
});