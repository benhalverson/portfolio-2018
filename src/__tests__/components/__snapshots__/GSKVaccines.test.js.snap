// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the GSKVaccines component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GSKVaccines />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="title"
>
          GSK Vaccines Eduaction Module
</div>,
        <div
          className="body"
>
          <p>
                    This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
          </p>
          <div
                    className="screenshots"
          >
                    <img
                              alt=""
                              className="col"
                              src="GSK001.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="GSK002.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="GSK003.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="GSK004.png"
                    />
                    <img
                              alt=""
                              className="col"
                              src="GSK005.png"
                    />
          </div>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "GSK Vaccines Eduaction Module",
          "className": "title",
        },
        "ref": null,
        "rendered": "GSK Vaccines Eduaction Module",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
</p>,
            <div
              className="screenshots"
>
              <img
                            alt=""
                            className="col"
                            src="GSK001.png"
              />
              <img
                            alt=""
                            className="col"
                            src="GSK002.png"
              />
              <img
                            alt=""
                            className="col"
                            src="GSK003.png"
              />
              <img
                            alt=""
                            className="col"
                            src="GSK004.png"
              />
              <img
                            alt=""
                            className="col"
                            src="GSK005.png"
              />
</div>,
          ],
          "className": "body",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
            },
            "ref": null,
            "rendered": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt=""
                  className="col"
                  src="GSK001.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="GSK002.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="GSK003.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="GSK004.png"
/>,
                <img
                  alt=""
                  className="col"
                  src="GSK005.png"
/>,
              ],
              "className": "screenshots",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "GSK001.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "GSK002.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "GSK003.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "GSK004.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "",
                  "className": "col",
                  "src": "GSK005.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="title"
>
            GSK Vaccines Eduaction Module
</div>,
          <div
            className="body"
>
            <p>
                        This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
            </p>
            <div
                        className="screenshots"
            >
                        <img
                                    alt=""
                                    className="col"
                                    src="GSK001.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="GSK002.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="GSK003.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="GSK004.png"
                        />
                        <img
                                    alt=""
                                    className="col"
                                    src="GSK005.png"
                        />
            </div>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "GSK Vaccines Eduaction Module",
            "className": "title",
          },
          "ref": null,
          "rendered": "GSK Vaccines Eduaction Module",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.
</p>,
              <div
                className="screenshots"
>
                <img
                                alt=""
                                className="col"
                                src="GSK001.png"
                />
                <img
                                alt=""
                                className="col"
                                src="GSK002.png"
                />
                <img
                                alt=""
                                className="col"
                                src="GSK003.png"
                />
                <img
                                alt=""
                                className="col"
                                src="GSK004.png"
                />
                <img
                                alt=""
                                className="col"
                                src="GSK005.png"
                />
</div>,
            ],
            "className": "body",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
              },
              "ref": null,
              "rendered": "This was a Web app based on an education framework called Adapt Learning. It was selected because of a really short deadline given by the client. The technology it used was Backbone.js and SCSS.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    className="col"
                    src="GSK001.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="GSK002.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="GSK003.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="GSK004.png"
/>,
                  <img
                    alt=""
                    className="col"
                    src="GSK005.png"
/>,
                ],
                "className": "screenshots",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "GSK001.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "GSK002.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "GSK003.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "GSK004.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "col",
                    "src": "GSK005.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
